app/src/ui/preferences/appearance.tsx

```diff
@@ -20,9 +20,8 @@ interface IAppearanceProps {
 const themes: ReadonlyArray<ISegmentedItem> = [
   { title: 'Light', description: 'The default theme of GitHub Desktop' },
   {
-    title: 'Dark (beta)',
-    description:
-      'A beta version of our dark theme. Still under development. Please report any issues you may find to our issue tracker.',
+    title: 'Dark',
+    description: 'GitHub Desktop is for you too, creatures of the night',
```

Comment:
The way I always thought of it?
```suggestion
    description: 'GitHub Desktop is for you too, children of darkness',
```

Reply:
Love the suggestion @say25, though I think I'd like to avoid allusions to things that might be not so pleasant if possible (in this case, the documentary film).

Reply:
haha I was thinking sort of `Chilling Adventures of Sabrina` but likely a good idea

Reply:
@say25 i'm with you! 😃

![sabrina spellman, winking](https://pmctvline2.files.wordpress.com/2018/10/chilling-adventures-of-sabrina-finale.jpg?w=620&h=420&crop=1)